holidays/financial/tel_aviv_stock_exchange.py

```
@@ -0,0 +1,211 @@
+        [2025](https://web.archive.org/web/20260813161220/https://content.tase.co.il/media/ob4blknd/file_0010_vacation_schedule_2025_eng.pdf)
+        [2026](https://web.archive.org/web/20260418172343/https://www.tase.co.il/en/content/knowledge_center/trading_vacation_schedule/)
+    """
+
+    market = "XTAE"
+    default_language = "he"
+    observed_label = "%s"
+    supported_categories = (PUBLIC, HALF_DAY)
+    supported_languages = ("en_US", "he")
+    start_year = 2013
+
+    def __init__(self, *args, **kwargs):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)